### PR TITLE
Fixed some grammatical errors and added entry on SonarQube

### PR DESCRIPTION
**Pull request by deerter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,5 +24,10 @@ Alberto Martinez Menendez | 681061 | [![Build Status](https://travis-ci.org/Belb
 Daniel Rueda Macías | 559207 | [![Build Status](https://travis-ci.org/danirueda/lab1-git-race.svg)](https://travis-ci.org/danirueda/lab1-git-race)
 Jose Angel Caudevilla Casaus | 649003 | [![Build Status](https://travis-ci.org/josnick/lab1-git-race.svg)](https://travis-ci.org/josnick/lab1-git-race)
 Daniel Barea López | 666156 | [![Build Status](https://travis-ci.org/dbarelop/lab1-git-race.svg)](https://travis-ci.org/dbarelop/lab1-git-race)
+<<<<<<< HEAD
```
Comment (RecuencoJones) [inline README.md]: you are pushing a conflict, please solve it

File: description.md [hunk 2]
```diff
@@ -304,6 +304,20 @@ These editors come bundled with native support for EditorConfig. Everything shou
...
+ - **Comments**
+ - **Coding rules**
+ - **Potential bugs**
+ - **Duplications**
+ - **Unit tests**
+ - **Complexity**
+ 
+The platform offers a visual report on the current projects and enables the user to replay the past so that he can follow the evolution in quality of his code.
+*SonarQube* covers more than twenty programming languages through plugins including Java, C++, C#, Cobol, Python, PHP among others.
+It can also be integrated into an IDE such as Eclipse, IntelliJ or Visual Studio through and extension called [SonarLint](http://www.sonarlint.org/) providing users with immediate feedback on quality issues and bugs injected into their code.
```
Comment (RecuencoJones) [inline description.md]: through an extension*

File: README.md [hunk 3]
```diff
@@ -24,10 +24,7 @@ Alberto Martinez Menendez | 681061 | [![Build Status](https://travis-ci.org/Belb
 Daniel Rueda Macías | 559207 | [![Build Status](https://travis-ci.org/danirueda/lab1-git-race.svg)](https://travis-ci.org/danirueda/lab1-git-race)
 Jose Angel Caudevilla Casaus | 649003 | [![Build Status](https://travis-ci.org/josnick/lab1-git-race.svg)](https://travis-ci.org/josnick/lab1-git-race)
 Daniel Barea López | 666156 | [![Build Status](https://travis-ci.org/dbarelop/lab1-git-race.svg)](https://travis-ci.org/dbarelop/lab1-git-race)
-<<<<<<< HEAD
 Álvaro Juan Ciriaco | 682531 | [![Build Status](https://travis-ci.org/aJuanCiri/lab1-git-race.svg)](https://travis-ci.org/aJuanCiri/lab1-git-race)
-yourname | your nia | your Travis-ci status
-=======
 Alvaro Barrios Gil | 666179 | [![Build Status](https://travis-ci.org/deerter/lab1-git-race.svg)](https://travis-ci.org/deerter/lab1-git-race)
 your name | your nia | your Travis-ci status
 >>>>>>> f6c474461e82b8088ad210679641167c9e3390f7
```
Comment (fjlopez) [inline README.md]: This line is the remaining of a conflict.

**Comment by fjlopez** [comment]
You must first sync your code with the upstream master as there are conflicting files (`README.md` and `description.md`)

**Pull request merged**